src/Compatibility/Microsoft.DotNet.PackageValidation/ReadMe.md

```diff
@@ -0,0 +1,198 @@
+## Package Validation
+
+With .NET Core & Xamarin we have made cross-platform a mainstream requirement for library authors. However, we lack validation tooling which can result in packages that don't work well which in turn hurts our ecosystem. This is especially problematic for emerging platforms where adoption isn't high enough to warrant special attention by library authors.
+
+The tooling we provide as part of the SDK has close to zero validation that multi-targeted packages are well-formed. For example, a package that multi-targets for .NET 6.0 and .NET Standard 2.0 needs to ensure that code compiled against the .NET Standard 2.0 binary can run against the binary that is produced for .NET 6.0. We have seen this issue in the wild, even with 1st parties, for example, the Azure AD libraries.
+
+Another common issue in the .NET ecosystem is that binary breaking changes aren't well understood. Developers tend to think of "does the source still compile" as the bar to determine whether an API is compatible. However, at the API level certain changes that are fine in C# aren't compatible at the level of IL, for example, adding a defaulted parameter or changing the value of a constant.
+
+Package Validation tooling will allow library developers to validate that their packages are well-formed and they they didn't make unintentional breaking changes.
+
+# How To Add Package Validation To Your Projects
+
+Package Validation is currently being shipped as a msbuild sdk package which can be consumed in the csproj. It is set of tasks and targets that run after the dotnet pack
+The csproj will look like
+
+```C#
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;net6.0</TargetFrameworks>
+  </PropertyGroup>
+
+  <Sdk Name="Microsoft.DotNet.PackageValidation" Version="1.0.0-preview4.21121.3" />
```

Comment:
is this the final preview version we are planning to go with? If not, should we not include Version and let MSBuild pick the latest one in NuGet? (Not sure if that works with preview versions)

Reply:
> is this the final preview version we are planning to go with? If not, should we not include Version and let MSBuild pick the latest one in NuGet? (Not sure if that works with preview versions)

we will probably need to specify it as we might be shipping multiple versions and latest version may not be the one that we want people to test. This version might change for the final post.